### PR TITLE
Add a note for travis-ci.com API switch

### PR DESCRIPTION
**Pull request by awarecan**
Ref: https://github.com/travis-ci/travis-ci/issues/10051#issuecomment-426228971

### CODE REVIEW

**Comment by AlmogCohen** [comment]
Wow....I've gave up a long time ago and your post [here](https://github.com/travis-ci/travis-ci/issues/10051) just fixed this...

I've never heard of the `--com` before nor was aware of it in anyway. To me the `.org` to `.com` change didn't quite register in anyway...I use both the same way...

**Comment by AlmogCohen** [comment]
Thank you!

**Comment by AlmogCohen** [comment]
Besides, this is their homegrown script... I would have expected them to check on the backend where this repo is being built currently and just provide the proper encrypted password...or give some warning to the user that they provide useless password.

File: user/deployment/pypi.md [hunk 1]
```diff
@@ -25,6 +25,12 @@ However, this would expose your PyPI password to the world. We recommend you
 travis encrypt your-password-here --add deploy.password
 ```
 
+If you are using travis-ci.com not travis-ci.org, you need add `--com` argument to switch travis API endpoint:
```
Comment (Pezi777) [inline user/deployment/pypi.md]:
```suggestion
If you are using travis-ci.com, not travis-ci.org, you need to add a `--com` argument to switch the Travis API endpoint:
```

**Pull request merged**